src/javascripts/components/example.mjs

```diff
@@ -1,22 +1,27 @@
 import iFrameResize from 'iframe-resizer/js/iframeResizer.js'
 
+/**
+ * Example component
+ *
+ * This allows automatic resizing of the iFrame pages contained within Example
+ * template wrappers.
+ *
+ * @param {Element} $module - HTML element to use for example
```

Comment:
Could potentially tighten this to [`HTMLIFrameElement`](https://developer.mozilla.org/en-US/docs/Web/API/HTMLIFrameElement), since we [only initialise this on iFrames](https://github.com/alphagov/govuk-design-system/blob/75c8b62ebb00f03f062d9eb491141ffbc6367967/src/javascripts/application.mjs#L24-L28).